### PR TITLE
tools: Generate config.js and secrets.json programmatically.

### PR DESCRIPTION
**Pull request by YagoGG**
This removes the last traces of the old long `cmd.get()` call, and avoids its usage for the sake of clarity and multiplaform support.

A possible followup could be replacing the command line `git` calls with [NodeGit](http://www.nodegit.org/).

### CODE REVIEW

File: tools/setup [hunk 1]
```diff
@@ -31,24 +32,41 @@ inquirer.prompt({
...
 }).then((answer) => {
   if (!answer.confirm) return;
-  console.log("Creating `src/config.js` and `src/secrets.json` file...");
-  cmd.get("echo '{\n  \"username\": \"zulipbot-test\",\n  \"password\": \"z#u%LIpB!OT11282016\",\n  \"zulip\": {\n    \"username\": \"zulipbot-bot@chat.zulip.org\",\n    \"apiKey\": \"G20349Kvd8OOb4ChmSsKhxQMsEyAWM7d\",\n    \"realm\": \"https:\/\/chat.zulip.org\"\n  }\n}' > src/secrets.json && ln -sf zulip_project_config.js src/config.js",
-    (err, data, stderr) => {
+  console.log("Creating `src/config.js` file...");
+  fs.link('src/zulip_project_config.js', 'src/config.js', (err) => {
+    if (err) {
+      if (err.code !== 'EEXIST') { throw err; }
+        console.log("There is already a file in `src/config.js`. Skipping.");
```
Comment (lonerz) [inline tools/setup]: Why is this indented?

Comment (YagoGG) [inline tools/setup]: Oh, that's a typo. Thanks!

**Comment by lonerz** [comment]
Will merge once I finish tests locally because I don't know why Travis is not running..

**Comment by lonerz** [comment]
Merged, thanks @YagoGG !

**Pull request closed without merging**